.circleci/verbatim-sources/linux-build-defaults.yml

```diff
@@ -37,6 +37,11 @@ pytorch_linux_build_defaults: &pytorch_linux_build_defaults
           NAMED_FLAG="export BUILD_NAMEDTENSOR=1"
         fi
 
+        # dispatch aten ops statically for mobile
+        if [[ ${BUILD_ENVIRONMENT} == *"android"* ]]; then
```

Comment:
we should also filter for iOS here, @xta0 - what is the right way to do it?

Reply:
I think we don't have ios CI yet.